src/overlays/actors/ovl_Obj_Lightswitch/z_obj_lightswitch.h

```diff
@@ -6,9 +6,42 @@
 
 struct ObjLightswitch;
 
+typedef void (*ObjLightswitchActionFunc)(struct ObjLightswitch*, GlobalContext*);
+
+enum ObjLightswitch_Type {
+    /*
+    doesn't turn off unless the switch flag is cleared some other way
+    */
+    OBJLIGHTSWITCH_TYPE_STAY_ON,
```

Comment:
Since these comments are brief, I think it would look cleaner if they just came after the enum entry on the same line